package.json

```diff
@@ -18,7 +18,13 @@
   },
   "type": "module",
   "bin": "./bin/lint-staged.js",
-  "exports": "./lib/index.js",
+  "exports": {
+    ".": {
+      "require": "./lib/index.js",
+      "import": "./lib/index.js"
+    },
```

Comment:
Since _lint-staged_ is pure ESM, should this be simply:

```suggestion
    ".": "./lib/index.js",
```

Reply:
@iiroj done